mu4/domain/notation/inotationstyle.h

```diff
@@ -0,0 +1,22 @@
+#ifndef INOTATIONSTYLE_H
+#define INOTATIONSTYLE_H
+
+#include <QVariant>
+
+#include "modularity/imoduleexport.h"
+#include "libmscore/style.h"
+
+using StyleId = Ms::Sid;
```

Comment:
Better put into `notationtypes.h`

Reply:
Yeah, I saw that file, but I prefer to leave it here because StyleId can be used only with "INotationStyle" interface

Reply:
I recommend (do not insist) still put in `notationtypes.h`, because:
* It is convenient to see all types (and exports from `libmscore` in this case) in one place.
* Currently used only here, but not known in the future
* This is a precedent, as a result, we can get a situation when types will be scattered from interface files and interfaces will be included for their use :)

Reply:
I've decided to move into notationtypes.h